pmlc/dialect/tile/tests/build-cion.mlir

```diff
@@ -16,17 +19,11 @@ func @dot(%arg0: tensor<1x2xf32>, %arg1: tensor<2x3xf32>) -> tensor<?x?xf32> {
   return %10 : tensor<?x?xf32>
 }
 
-// CHECK: #[[MAP0:map[0-9]+]] = affine_map<(d0, d1, d2) -> (d0, d1)>
-// CHECK: #[[MAP1:map[0-9]+]] = affine_map<(d0, d1, d2) -> (d0, d2)>
-// CHECK: #[[MAP2:map[0-9]+]] = affine_map<(d0, d1, d2) -> (d2, d1)>
```

Comment:
The specific structure of the affine maps is encoding some semantic information, it seems a little weird to remove these checks. That said, if that's the typical MLIR style, then I don't object.

Reply:
Not removed, just moved, look up above at line 3.

Reply:
whoops, thanks!